rackcity/api/serializers/it_model_serializers.py

```diff
@@ -5,7 +5,9 @@
 
 
 class ITModelSerializer(serializers.ModelSerializer):
-
+    height = RCIntegerField(
+        allow_null=True, max_value=2147483647, min_value=0, required=False
+    )
```

Comment:
Can you explain why you added this to the serializer?

Reply:
Yep - we use RCIntegerField (rather than the default serializers.IntegerField) for all optional numeric fields to handle empty strings being passed from forms